src/fluent_api/mod.rs

```diff
@@ -41,33 +41,41 @@ where
         Self { db }
     }
 
+    /// Select a document from the database
```

Comment:
This is not just one document. It can select documents using query filter or particular documents using their IDs.